generator.rb

```diff
@@ -0,0 +1,67 @@
+# WAVE ONE/TWO
+# is_vowel? 
+def is_vowel?(letter)
+    if letter == "a" || letter == "e" || letter == "i" || letter == "o" || letter == "u"
+        return true
+    else
+        return false
+    end
+end
+
+# run generator
+def run_generator #this is a method
+# this is a test to make sure it worked properly:
+    #   test_vowel = "i"
+    #   puts "Is #{test_vowel} a vowel?"
+    #   puts is_vowel?(test_vowel)
+
+    #   test_consonant = "g"
+    #   puts "Is #{test_consonant} a vowel?"
+    #   puts is_vowel?(test_consonant)
+
+# per the instructions in wave one
+    puts "what is the first word to combine?"
+    first_word = gets.chomp
+    puts "what is the last work to combine?"
+    second_word = gets.chomp
+    puts "Your first word is #{first_word}"
+    puts "Your second word is #{second_word}"
+    puts "in the run_generator method"
+
+# settiing up the index/dummy word [word1_end] to push values into
+# using .reverse to count the first vowel (last vowel) and .each_char
+    index = 0
+    word1_end = nil
+    (first_word.reverse).each_char do |letter|
```

Comment:
Nice, I love that you used parentheses to make the grouping more clear `(first_word.reverse)`